Travel Scheduler/View Controllers/MoreOptionViewController.m

```diff
@@ -87,7 +92,63 @@ - (void)createCollectionView
     [self.collectionView setBackgroundColor:[UIColor whiteColor]];
     [self.view addSubview:self.collectionView];
 }
+    
+- (void)loadMoreData {
+    NSString *correctType;
+    if([self.stringType isEqualToString:@"Hotels"]) {
+        correctType = @"lodging";
+    } else if ([self.stringType isEqualToString:@"Restaurants"]) {
+        correctType = @"restaurant";
+    } else {
+        correctType = @"park";
+    }
+    
+    [self.hub updateArrayOfNearbyPlacesWithType:correctType withCompletion:^(bool success, NSError * _Nonnull error) {
+        if(success) {
+            self.places = self.hub.dictionaryOfArrayOfPlaces[correctType];
+        }
+        else {
+            NSLog(@"did not work");
+        }
+        self.isMoreDataLoading = NO;
+        [loadingMoreView stopAnimating];
+        [self.collectionView reloadData];
+    }];
+}
+
+#pragma mark - Infinite scrolling helper methods
+
+- (void) setUpInfiniteScrollIndicator
```

Comment:
Uh... I don't think there should be a space after (void)?

Reply:
You're right! Gonna fix that

Reply:
Done!